src/python_testing/TC_SEAR_1_6.py

```diff
@@ -72,7 +72,7 @@ async def read_progress(self, step):
         return progress
 
     def TC_SEAR_1_6(self) -> list[str]:
-        return ["SEAR.S", "SEAR.S.A0005", "SEAR.S.A0000", "SEAR.S.A0002", "SEAR.S.M.HAS_MANUAL_OPERATING_STATE_CONTROL"]
+        return ["SEAR.S", "SEAR.S.M.HAS_MANUAL_OPERATING_STATE_CONTROL"]
```

Comment:
Actually, let's just leave this test as-is for now. The top level pics we can deal with separately and this will mess up the TH

Reply:
Thank you,
This has been updated to leave the top level PICS as-is and should now be resolved.